fs-person/fs-person.html

```diff
@@ -27,6 +28,11 @@
       :host([inline]) .fs-person-vitals__name {
         margin-right: 15px;
       }
+      .fs-person-vitals,
+      .fs-person-vitals__name-block {
+        width: 98%;
+        width: calc(100% - 1.143rem);
```

Comment:
I am open to suggestions. The reason this is not 100% is because of the commonly preceding `.fs-person-gender__icon` dot, whose size and margin screw up the 100% width calculation.